youtube_dl/extractor/rockstargames.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    qualities,
+    parse_iso8601
+)
+
+
+class RockstarGamesIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?rockstargames\.com/videos/video/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://www.rockstargames.com/videos/video/11544/',
+        'md5': '03b5caa6e357a4bd50e3143fc03e5733',
+        'info_dict': {
+            'id': '11544',
+            'ext': 'mp4',
+            'title': 'Further Adventures in Finance and Felony Trailer',
+            'thumbnail': 're:^https?://.*\.jpg$',
+            'description': """Build an empire as a CEO. Traffic illicit cargo and contraband. Disrupt rival supply chains. Acquire extravagant new vehicles. Open prestigious new offices. Hire an executive assistant. And run the town, one hostile takeover at a time.""",
+            'upload_date': '20160602',
+            'timestamp': 1464876000
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        json_url = 'https://www.rockstargames.com/videoplayer/videos/get-video.json?id=%s&locale=en_us' % (video_id)
```

Comment:
Parenthesis are superfluous. Move `json_url` right into `_download_json` call - it's only used once.